### DISCUSSION TITLE
Crossband Repeater

### DISCUSSION THREAD

**Discussion by yprat** [egzumer/uv-k5-firmware-custom#329] [Ideas]
Is it possible to add crossband repeater?

**Comment by ErikS-web** [comment]
I don't think so.... to my opinion the hardware does not support.

[Menu 59,](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Menu#main-menu) describes the supported modes.

**Comment by ErikS-web** [comment]
There is a hardware solution. A repeater-box connected to 2 radio's.

**Comment by yprat** [comment]
My idea was that if the hardware could support it, it would be nice to have.

**Comment by egzumer** [comment]
Voice repeater with one radio - no. Digital repeater yes. Even aprs repeater should be possible.

**Comment by AndrewTXJ** [comment]
definitely nope, it's just a $18.59 radio (as the date i look on ebay [12/31/2023] ), don't expect to much on its hardware🤣🤣🤣, its not a duplex radio, you can't receiving while transmitting at the same time, buy a proper repeater, or a HT that support this function like some wouxun HT, or you can buy a cheap repeater box, and buy two cheap radio that have dualband like the baofeng UV-5R, i have one, work so terribly or not working at all, but you can still try it out, maybe i brought a defective one, there is a bunch of those sell online, your mileage may vary.

**Reply by yprat** [reply]
I have it already on my IC 2730 and ft8800 so i don't need any extra. I just wonder if it was possible with firmware update on uv k5.

**Reply by AndrewTXJ** [reply]
100% nope, so sorry
